### ISSUE TITLE
Prints warnings abut missing .eslintrc.js and "undefined" on console

### ISSUE THREAD

**Issue by realh** [sammydre/ts-for-gjs#29]
Each time it completes processing a module ts-for-gjs prints "undefined" on the console. I haven't managed to find out what's causing this.

**Comment by realh** [comment]
I was going to report a separate issue, but they might both be caused by the same thing. Just before each "undefined" message I get a pair of warnings about ESLint like this:

```
Error: No ESLint configuration found in /home/tony/Code/tvfrontend/gjs/tsforgjs/@types/Gjs.
[gjs][GLib-2.0] WARN: Can't prettify file: "@types/Gjs/GLib-2.0.d.ts", please check your .eslintrc.js in your working directory
```

Could the program check a .eslintrc.js exists before trying to run ESLint, or perhaps provide a suitable template?

**Comment by JumpLink** [comment]
I think I can fix this when I am back from vacation

**Comment by JumpLink** [comment]
Should be fixed in #45
